source/src/data-mapper/Formatter/DateTimeFormatter.cs

```diff
@@ -0,0 +1,31 @@
+namespace Buhlergroup.DataMapper.Formatter
+{
+    using System;
+    using System.Globalization;
+    using Buhlergroup.DataMapper.Model;
+
+    public static class DateTimeFormatter
+    {
+
+        public static string Format(FieldMappingModel mapping, object value)
+        {
+            if (String.IsNullOrEmpty(mapping.InFormat))
+            {
+                return value.ToString();
+            }
+
+            return DateTime.ParseExact(value.ToString(), mapping.InFormat, CultureInfo.InvariantCulture, DateTimeStyles.None).ToString(mapping.OutFormat);
```

Comment:
```suggestion
            return DateTime.ParseExact(value.ToString(), mapping.InFormat, CultureInfo.InvariantCulture, DateTimeStyles.None).ToString(mapping.OutFormat, CultureInfo.InvariantCulture);
```
To prevent inconsistent results from CultureInfo